src/dpapi_ng/_blob.py

```diff
@@ -150,6 +150,43 @@ class DPAPINGBlob:
     enc_content_algorithm: str
     enc_content_parameters: t.Optional[bytes]
 
+    def pack(self, sid:str) -> bytes:
+        # TODO: it's not very nice to pass sid as separate parameter here, should be extracted from self.security_descriptor
```

Comment:
It's probably worth splitting out `security_descriptor` in the future so it can better handle packing different types but as this is internal we can definitely look at that later and just keep it as you've done it for now.